src/org/omegat/Bundle.properties

```diff
@@ -82,75 +82,100 @@ app-version-template-pretty={0} {1}
+    \t  Disable team project functionality.\n\
+    \tteam init <source language> <target language>\n\
+    \t  Team project tool.\n\
+    \t  Initialize a minimal team project with the specified languages.\n\
+    \t  If invoked in a git working tree or svn checkout, the results\n\
+    \t  will be staged/added.\n\
+\n\
+    \t--disable-project-locking\n\
+    \t  Do not attempt to lock the project's omegat.project file.\n\
+    \t--disable-location-save\n\
+    \t  Do not remember the last directory opened in the file picker.\n\
+\n\
+    \t--ITokenizer=<classname>\n\
+    \t--ITokenizerTarget=<classname>\n\
+    \t  Source- or target-language tokenizer to use (overrides project\n\
+    \t  settings). See OmegaT.jar/META-INF/MANIFEST.MF for valid values.\n\
+\n\
+    \t--config-file=<path>\n\
+    \t  A Java .properties file from which to load default settings.\n\
+    \t--config-dir=<path>\n\
+    \t  The directory to read/write OmegaT configuration files from/to.\n\
+    \t--resource-bundle=<path>\n\
+    \t  A Java .properties file to use for interface text\n\
+\n\
+    \t--mode=[console mode name]\n\
+    \t  Specify a mode other than default (GUI). See below for details.\n\
+\n\
+Console mode names and their options:\n\
+\n\
+    \t--mode=console-translate\n\
+    \t  --source-pattern=<pattern>\n\
+    \t    A regex whitelist of source files to process.\n\
+\n\
+    \t--mode=console-pseudotranslatetmx\n\
+    \t  --pseudotranslatetmx=<path>\n\
+    \t    The output pseudotranslated TMX file.\n\
+    \t  --pseudotranslatetype=[equal|empty]\n\
+    \t    What to fill the pseudotranslated TMX with.\n\
+\n\
+    \t--mode=console-align\n\
+    \t  --alignDir=<path>\n\
+    \t    A directory containing the translated files.\n\
+\n\
+    \t--mode=console-stats\n\
+    \t  project_path\n\
+    \t    The path to the project\n\
+    \t  --output-file=[stats-output-file]\n\
+    \t    Prints to that file, or to standard output if absent.\n\
+    \t    Without --stats-type, detects format from file extension.\n\
+    \t    Defaults to xml.\n\
+    \t  --stats-type=[xml|text][txt][json]]]\n\
+    \t    Specifies the output format.\n\
+\n\
+Console only options:\n\
+\n\
+    \t--quiet\n\
+    \t  Reduce output to command line stdout.\n\
+    \t--script=<path>\n\
+    \t  A script file to be executed on project events.\n\
+    \t--tag-validation=[abort|warn]\n\
+    \t  Validate tags. Abort: Exit with an error if tags invalid.\n\
+    \t  Warn: Report invalid tags to stdout.\n\
+\n\
+{0}
+
+#TEAM_TOOL_HELP=Usage: java -jar OmegaT.jar team <command> [<options>]]\n\
+#    \n\
+#    Commands:\n\
+#        \t-h, --help\n\
+#            \t\tShow usage information.\n\
+#        \tinit <source language> <target language>\n\
+#            \t\tInitialize a minimal team project with the specified languages.\n\
+#            \t\tIf invoked in a git working tree or svn checkout, the results\n\
+#            \t\twill be staged/added.\n\
+#    \n\
+#    {0}
```

Comment:
```suggestion
TEAM_TOOL_HELP=Usage: java -jar OmegaT.jar team <command> [<options>]]\n\
    \n\
    Commands:\n\
        \t-h, --help\n\
            \t\tShow usage information.\n\
        \tinit <source language> <target language>\n\
            \t\tInitialize a minimal team project with the specified languages.\n\
            \t\tIf invoked in a git working tree or svn checkout, the results\n\
            \t\twill be staged/added.\n\
    \n\
    {0}
```

Reply:
I don't think it is necessary since I put the explanations between line 104 and 108.

Reply:
CI test failed with `java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key TEAM_TOOL_HELP`

Reply:
Ok. Sorry. I'll looking for the place in the code where this is expected. Unless you have a strong opinion about that help key, I'd rather remove it from the code.